addons/laserpointer/functions/fnc_getNearUnits.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: commy2
+ * Reports near units.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _camPosAGL = positionCameraToWorld [0, 0, 0];
+
+// handle RHS / bugged vehicle slots
+if !((_camPosAGL select 0) isEqualType 0) exitWith {};
```

Comment:
I guess. I have an (probably irrational) disdain for `isEqualTypeArray` and `isEqualTypeParams`, because of the strange ways they handle nil and different sized arrays.